extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -265,15 +249,11 @@ export class ImageViewer {
    * @param {?Element} sourceImage
    */
   init(sourceElement, sourceImage) {
-    this.sourceWidth_ = this.getSourceWidth_(sourceElement, sourceImage);
-    this.sourceHeight_ = this.getSourceHeight_(sourceElement, sourceImage);
+    this.setSourceDimensions_(sourceElement, sourceImage);
     this.srcset_ = srcsetFromElement(sourceElement);
 
-    Object.keys(this.ariaAttributes_).forEach(key => {
-      this.ariaAttributes_[key] = sourceElement.getAttribute(key);
-      if (this.ariaAttributes_[key]) {
-        this.image_.setAttribute(key, this.ariaAttributes_[key]);
-      }
+    sourceElement.getImpl().then(e => {
```

Comment:
nit: `e` is a little too ambiguous for this variable name. Is it the `BaseElement` subclass instance? Usually `e` is just for `Event` instances.